src/hooks/protections/receive_net_message.cpp

```diff
@@ -235,7 +236,7 @@ namespace big
 			switch (msgType)
 			{
 			case rage::eNetMessage::MsgScriptMigrateHost: return true;
-			case rage::eNetMessage::MsgRadioStationSyncRequest: return true;
+			case rage::eNetMessage::MsgRadioStationSyncRequest: return false;
```

Comment:
Now we will be susceptable to OOM kicks. You have to address both scenarios.

Reply:
I will do this.

Reply:
To be fair, there was no comment at all for why that was implemented. There probably should've been a comment there explaining that it patches a certain crash/kick.